go/libraries/doltcore/env/actions/remotes.go

```diff
@@ -494,3 +495,30 @@ func HandleInitRemoteStorageClientErr(name, url string, err error) error {
 	var detail = fmt.Sprintf("the remote: %s '%s' could not be accessed", name, url)
 	return fmt.Errorf("%w; %s; %s", ErrFailedToGetRemoteDb, detail, err.Error())
 }
+
+func ParseRemoteBranchName(startPt string) (string, string) {
+	startPt = strings.TrimPrefix(startPt, "remotes/")
+	names := strings.Split(startPt, "/")
```

Comment:
(minor) this could be a good place to use `strings.SplitN` and have it split into (at most) 2 parts; then you wouldn't have to rejoin the other strings together.